index.css

```diff
@@ -483,6 +484,100 @@ q:before, q:after{ content: "";}
     cursor: pointer;
 }
 
+/* surf camps - first article */
+
+.surf-camps {
+    position: relative;
+    margin-bottom: 120px;
+}
+
+.surf-camps__article {
+    position: absolute;
+}
```

Comment:
No need for absolute positioning here. Just switch the order of the `<article>` and the `<img>` elements.